### PR TITLE
Let's try this again. PDF Generator Class

### PR DESCRIPTION
**Pull request by jmonson2**

### CODE REVIEW

File: PDF.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	private PDFont font = PDType1Font.TIMES_ROMAN;
+	private PDPage page = new PDPage();
+	private ResultSet rs;
+	private Connection con;
+	private final String conURL; //URL for connecting to DB
+	private String conUSER;//Username for connecting to DB
+	private String conPW;//Password for connecting to DB
+	private String filePath;//Path to save the pdf
+	
+	public PDF(String conURL, String conUSER, String conPW, String filePath) {
```
Comment (kabollinger03) [inline PDF.java:24]: Why do we need connections and all that stuff? I thought we decided that we are just storing it on the local machine.

**Review by kabollinger03 — APPROVED** [review]
Looks good. Didn't look too closely since your comments seem like you are aware of the current problems. I would rather have working code with documented shortcomings than nobody knowing how stuff is done.

File: PDF.java (hunk 1 above)
Comment (jmonson2) [reply PDF.java:24]: This is to just pull data from database. Not to store the pdf.

**Review by ChrisSMendoza — APPROVED** [review]
When it comes time to integrate we'll probably grab DB data elsewhere then pass those results into this class to generate the PDF. Solid job John.

**Pull request merged**